app/uk/gov/hmrc/gatekeeper/controllers/ApplicationController.scala

```diff
@@ -143,6 +143,14 @@ class ApplicationController @Inject() (
       ColumnDefinition("Status", (app => app.state.name.displayText)),
       ColumnDefinition("Rate limit tier", (app => app.details.rateLimitTier.toString())),
       ColumnDefinition("Access type", (app => app.access.accessType.toString())),
+      ColumnDefinition(
+        "Overrides",
+        app =>
+          app.access match {
+            case Access.Standard(_, _, _, overrides, _, _) => overrides.mkString(",")
```

Comment:
Won't this mess up the CSV if there's more than one override???

Reply:
Nah, the csv writer properly wraps it in quotes to let it be read in as text.